### PR TITLE
Docs: Add release notes for 2.7.4

### PR DESCRIPTION
**Pull request by DylanGuedes**
**What this PR does / why we need it**:
Add release notes for v2.7.4

### CODE REVIEW

**Review by JordanRushing — APPROVED** [review]
Looks great; thanks for putting this together!

File: docs/sources/release-notes/v2-7.md [hunk 1]
```diff
@@ -33,6 +33,18 @@ As always, please read the [upgrade guide](../../upgrading/#270) before upgradin
...
+### 2.7.4 (2023-02-24)
+
+* Fixed different streams for `cri` tags ending on the same stream
+* Fixed the `userdata` field (from Windows Event Log) being scrapped wrongly
+* Fixed `vector()` function producing wrong timestamp
+* Fixed behavior for overlapping chunks with multiple stores
+* Fixed logs results caching causing query-frontend to return logs outside of query window
+* Fixed panics when:
+  * `/scheduler/ring` endpoint is requested with scheduler ring disabled
+  * LogQL clones a especific query
```
Comment (JordanRushing) [inline docs/sources/release-notes/v2-7.md:45]:
```suggestion
  * LogQL clones a specific query
```

File: docs/sources/release-notes/v2-7.md [hunk 2]
```diff
@@ -33,6 +33,18 @@ As always, please read the [upgrade guide](../../upgrading/#270) before upgradin
 
 ## Bug fixes
 
+### 2.7.4 (2023-02-24)
+
+* Fixed different streams for `cri` tags ending on the same stream
+* Fixed the `userdata` field (from Windows Event Log) being scrapped wrongly
```
Comment (JordanRushing) [inline docs/sources/release-notes/v2-7.md:39]:
```suggestion
* Fixed the `userdata` field (from Windows Event Log) being scraped incorrectly
```

**Review by JStickler — APPROVED** [review]
[Docs squad]  LGTM

**Pull request merged**